### PR TITLE
Textbox overflow on copy/paste of bigger text 

### PR DESCRIPTION
**Pull request by Culprit-CZ**
It is problem with `.sn-us-mentions`. It's height needs to be updated along with updating height of the textarea.

http://phpbbsocialnetwork.com/viewtopic.php?f=44&t=4307

### CODE REVIEW

**Comment by senky** [comment]
Along with `sn-theme/prosilver/classes_userstatus.less`, `root/styles/prosilver/theme/socialnet.css` must be updated and sent to this PR.

File: sn-theme/prosilver/classes_userstatus.less [hunk 1]
```diff
@@ -178,9 +178,11 @@
   .sn-us-mentions{
   	top: 1;
```
Comment (senky) [inline sn-theme/prosilver/classes_userstatus.less]: As you state `top` attribute in line 184, you can delete this one.

**Comment by senky** [comment]
I have tested it, and it works, however I quite do not understand why. Can you explain me, how can negative `top` and `left` cause, that text does not overflows any more?

**Comment by Culprit-CZ** [comment]
How to explain?
Elastic (expanding textarea) plugin create a `div` that is placed before.
The position of this `div` was same as position for the `textarea`.
For right count of the height should be not `hidden`.

File: sn-theme/prosilver/classes_userstatus.less [hunk 2]
```diff
@@ -198,6 +198,7 @@
       color: #fff;
       white-space: pre-wrap;
       width: 100%;
+
```
Comment (senky) [inline sn-theme/prosilver/classes_userstatus.less]: This line should not be here, I believe.

**Comment by senky** [comment]
Thanks for your explanation! I like you solved it without changing Javascript :)

File: sn-theme/prosilver/classes_userstatus.less (hunk 2 above)
Comment (Culprit-CZ) [inline sn-theme/prosilver/classes_userstatus.less]: It is irrelevant, It is our dev code.
No problem I will remove.

**Comment by Culprit-CZ** [comment]
I'm glad that it could be solved without JavaScript

**Comment by senky** [comment]
MA

**Comment by Culprit-CZ** [comment]
@Senky please control my squash

**Comment by senky** [comment]
Perfect!

MA

**Pull request merged**